config/local.json-dist

```diff
@@ -0,0 +1,8 @@
+{
+  "publicUrl": "http://127.0.0.1:1114",
+  "env": "dev",
+  "log": {
+    "format": "pretty",
```

Comment:
The default is [pretty](https://github.com/mozilla/fxa-basket-proxy/pull/2/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R22), so this shouldn't be needed.